Update docker-aspnetmvc.md (#9634)
aspnet/Docs: aspnet/mvc/overview/deployment/docker-aspnetmvc.md

## Build the image

> You will need to define your Docker image in a Dockerfile, which literally means that to create a text file and to name it dockerfile. The Dockerfile contains instructions to build the final image and includes information of the base image, additional components, the app you want to run, and other configuration images.  The Dockerfile is the input to the `docker build` command, which creates the image.

Reviewer: ```suggestion
Create a new file named *Dockerfile* to define your Docker image. *Dockerfile* contains instructions to build the final image and includes any base image names, required components, the app you want to run, and other configuration images. *Dockerfile* is the input to the `docker build` command that creates the image.
```
